.gitignore

```diff
@@ -144,3 +144,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.DS_Store
```

Comment:
This is an OS-specific file, not a project-specific file. I usually add those to global gitignores (`~/.gitignore`), not project gitignores.